webclient/components/core/search-phrases/call-insert-search-phases-of-components-and-handle-selection.vue

```diff
@@ -86,6 +87,7 @@ import insertSsComponentSearchPhrases from '@/components/pt-info/single/1time-Mr
 
 // 1time-Mrow-mField
 import insertPhoneNumberComponentSearchPhrases from '@/components/pt-info/single/1time-Mrow-mField/phone-numbers/static-data/insert-into-master-of-search-phrases-ct.vue'
+import insertDignosisComponentSearchPhrases from '@/components/pt-info/single/1time-Mrow-mField/diagnosis/db/client-side/static-data/search-phrases-ct.vue'
```

Comment:
File name should be 'insert-into-master-of-search-phrases-ct.vue' instead of 'search-phrases-ct.vue'